docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,47 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 10 May 2023
+
+Frontend version: .0.0-20230509.1043 | Backend version: 1.61.9-230508.0923
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Enhancement to SEED device onboarding Status for public officers.
+</summary>
+
+Previously, the device status was displayed as **pending** until the device was successfully onboarded. However, this status did not provide Public Officers with much visibility into the onboarding process.
+
+We have now updated the device status to include more detailed information. The new statuses are as follows:
+
+- Triggered: Indicates that the onboarding process has been triggered for the device.
+- Software installed: Indicates that the software has been installed and the device is awaiting backend onboarding.
+- Failed: Indicates that the onboarding process has failed. In this case, the user will be able to see the reason for the failure and can click on the **retry** button to re-trigger onboarding. Depending on the reason for failure, retry may or may not be possible. If it is not possible, the 'retry' button will not be displayed.
+
+> **Note**: This change only affects Public Officers. Vendors currently do not trigger onboarding through the TechPass portal and do not have a **pending** device.
+
+</details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> TechBiz groups are excluded from <i>ListGroups</i> implementation by default.</summary>
+
+TechBiz groups will be hidden by default on TechPass portal. This change is also consistent with TechPass portal's **Groups** page, as we are not displaying the TechBiz groups there. However, if you want to include TechBiz groups via the Automation API, you will need to pass the query parameter <i>includeOnTechBiz=true</i>.
+
```

Comment:
TechBiz group is already hidden by default on TP portal currently, so this is not a new change in this release. we can just say:

Groups managed by TechBiz are now excluded from <i>ListGroups</i> Automation API by default. To include them, you will need to pass the query parameter <i>includeOnTechBiz=true</i>